roamance/src/components/auth/AuthForm.tsx

```
@@ -0,0 +1,35 @@
+  subtitle: string;
+}
+
+export function AuthForm({ children, title, subtitle }: AuthFormProps) {
+  return (
+    <motion.div
+      initial={{ opacity: 0, x: 20 }}
+      animate={{ opacity: 1, x: 0 }}
+      transition={{ duration: 0.6, delay: 0.3 }}
+      className="flex flex-col space-y-6 p-8 bg-card/30 backdrop-blur-xl rounded-2xl border border-border/50 shadow-lg relative overflow-hidden"
+    >
+      <div className="absolute inset-0 bg-gradient-to-br from-primary/5 via-transparent to-forest/5 pointer-events-none"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)